.storybook/components/PageTemplate.tsx

```diff
@@ -1,14 +1,14 @@
 import { PropsWithChildren, useMemo } from 'react';
-import { Logo } from '@ag.ds-next/react/ag-branding';
-import { Box } from '@ag.ds-next/react/box';
-import { Stack } from '@ag.ds-next/react/stack';
-import { tokens } from '@ag.ds-next/react/core';
-import { Footer, FooterDivider } from '@ag.ds-next/react/footer';
-import { Header } from '@ag.ds-next/react/header';
-import { AvatarIcon } from '@ag.ds-next/react/icon';
-import { LinkList } from '@ag.ds-next/react/link-list';
-import { MainNav } from '@ag.ds-next/react/main-nav';
-import { Text } from '@ag.ds-next/react/text';
+import { Logo } from '../../packages/react/src/ag-branding';
```

Comment:
I'm not sure if I'm happy with this. but it does work. docs and stuff use the package import and work fine, so I feel this should too. I tried aliases in the vite config, but couldn't get them to work.

Reply:
Yeah, fair enough. I think this is alright. Perhaps we can revisit after a Storybook 8 upgrade and Vite 11 upgrade?

Reply:
Yeah, I'm okay with that if you wan to approve